src/test/java/api/loans/DueDateScheduledNoticesTests.java

```diff
@@ -544,4 +549,104 @@ public void noticesShouldBeRescheduledAfterManualDueDateChange() {
       .until(scheduledNoticesClient::getAll, scheduledNoticesAfterRecallMatcher);
     assertThat(scheduledNoticesClient.getAll(), hasSize(6));
   }
+
+  @Test
+  public void scheduledOverdueNoticesShouldBeDeletedAfterOverdueFineIsCharged() {
+    useFallbackPolicies(loanPoliciesFixture.canCirculateRolling().getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandardDoNotCountClosed().getId(),
+      lostItemFeePoliciesFixture.facultyStandard().getId());
+
+    UUID uponAtTemplateId = UUID.randomUUID();
+
+    UUID afterTemplateId = UUID.randomUUID();
+    Period afterPeriod = Period.days(3);
+    Period afterRecurringPeriod = Period.hours(4);
+
+    JsonObject uponAtDueDateNoticeConfiguration = new NoticeConfigurationBuilder()
+      .withTemplateId(uponAtTemplateId)
+      .withDueDateEvent()
+      .withUponAtTiming()
+      .sendInRealTime(false)
+      .create();
+    JsonObject afterDueDateNoticeConfiguration = new NoticeConfigurationBuilder()
+      .withTemplateId(afterTemplateId)
+      .withDueDateEvent()
+      .withAfterTiming(afterPeriod)
+      .recurring(afterRecurringPeriod)
+      .sendInRealTime(true)
+      .create();
+
+    NoticePolicyBuilder noticePolicy = new NoticePolicyBuilder()
+      .withName("Policy with due date notices")
+      .withLoanNotices(Arrays.asList(
+        uponAtDueDateNoticeConfiguration,
+        afterDueDateNoticeConfiguration));
+    use(noticePolicy);
```

Comment:
Can it be removed and the above `useFallbackPolicies` call is executed at this line?

Reply:
Done (calling `useFallbackPolicies` now with this particular `noticePolicy`).